bin/tbg-submit

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+#
+# Copyright 2013-2018 Axel Huebl, Rene Widera, Richard Pausch
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+# This script parses template files for a batch systems and fills them
+# with a platform-independent configuration, which can then be submitted
+# to a batch system or plainly executed. It can also execute additional
+# commands at that point, e.g. to copy and archive input files alongside
+# a batch job.
+
+function absolute_path()
+{
+    cd $1
+    pwd
+}
+
+help()
+{
+    echo "TBG-submit (template batch generator)"
+    echo "submit a job created by tbg"
+    echo ""
+    echo "usage: tbg-submit [-s submitcommand] [-h] jobDirectory"
```

Comment:
I would add `[-t template]` in case one wants to change it

Reply:
Than I need to run tbg again. For that case you should use `tbg` directly.
The tool from the PR is not doing any substitution.

Reply:
That might be a misunderstanding.
I want like you that a user can run
```bash
pic-submit /job/dir
```

You allow to optionally set `-s qsub` and we can also add optionally `-t etc/picongpu/abc/some.tpl`.

Reply:
Yes `-s` is is only the explicit way to set the batch system. This is the pro user option.
`As I wrote `-t` needs to call all substitutions again. If I like to have it I need to run `tbg` again.
This is not a wrapper around `tbg`.
`-t` would also mean I need to update the `tpl` file from the user in the simulation folder. This has to much side effects and is hard to explain.
Do not forget you can use `tbg -s -t YETANOTHER.tpl -c PATH/tbg/submit.cfg -f PATH`

Reply:
Oh, too bad. I though we write a tbg wrapper.